ci/ci/build.py

```diff
@@ -352,15 +352,37 @@ def build(self, batch, code, scope):
         )
 
     def cleanup(self, batch, scope, parents):
-        if CLOUD == 'azure':
-            log.warning('Image cleanup in ACR is not yet supported')
-            return
-        assert CLOUD == 'gcp'
-
         if scope == 'deploy' and self.publish_as and not is_test_deployment:
             return
 
-        script = f'''
+        if CLOUD == 'azure':
+            image = 'mcr.microsoft.com/azure-cli'
+            assert self.image.startswith(DOCKER_PREFIX)
+            image_name = self.image[len(f'{DOCKER_PREFIX}/') :]
```

Comment:
formatting here? Should there be the extra space before the colon?

Reply:
yes, `black` put it there.